src/main/java/fi/metatavu/dcfb/server/search/index/IndexableItem.java

```diff
@@ -37,40 +38,44 @@
   @Field(analyzer = "english")
   private List<String> descriptionEn;
 
-  @Field(index = "not_analyzed", store = true)
+  @Field(type="keyword", store = true)
   private UUID categoryId;
 
-  @Field(index = "not_analyzed", store = true)
+  @Field(type="keyword", store = true)
   private UUID locationId;
 
-  @Field(index = "not_analyzed", store = true)
+  @Field(type="keyword", store = true)
   private String slug;
+  
+  @Field(type="geo_point")
+  private GeoPoint geoPoint;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename field "geoPoint" to prevent any misunderstanding/clash with field "GEOPOINT" defined on line 21. ([squid:S1845](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1845))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=21&issues=AWVnq7ubXKENQn1Shuhs&open=AWVnq7ubXKENQn1Shuhs)

[ISSUE_KEY]: AWVnq7ubXKENQn1Shuhs